.github/workflows/notebook-test-cron.yml

```
@@ -29,7 +29,23 @@ jobs:
           instance: "client-enablement/documentation/qiskit-documenta"
 
       - name: Execute notebooks
-        run: tox -- --only-submit-jobs
+        timeout-minutes: 330
+        run: tox -- --write --only-submit-jobs
+
+      - name: Detect changed notebooks
+        id: changed-notebooks
+        if: "!cancelled()"
+        run: |
+          echo "CHANGED_NOTEBOOKS<<EOF" >> $GITHUB_OUTPUT
+          echo "$(git diff --name-only)" >> $GITHUB_OUTPUT
```

Nit: this should work the same:

```suggestion
          git diff --name-only >> $GITHUB_OUTPUT
```